CPAC/resources/templates/BIDS_identifiers.tsv

```diff
@@ -20,10 +20,10 @@
 /ndmg_atlases/label/Human/HarvardOxfordcort-maxprob-thr25_space-MNI152NLin6_res-2x2x2.nii.gz	HOCPA	th25
 /ndmg_atlases/label/Human/HarvardOxfordsub-maxprob-thr25_space-MNI152NLin6_res-2x2x2.nii.gz	HOSPA	th25
 /ndmg_atlases/label/Human/Juelich_space-MNI152NLin6_res-2x2x2.nii.gz	Juelich	
-/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-200Parcels17NetworksOrder.nii.gz	Schaefer2018	200p17n
-/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-300Parcels17NetworksOrder.nii.gz	Schaefer2018	300p17n
-/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-400Parcels17NetworksOrder.nii.gz	Schaefer2018	400p17n
-/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-1000Parcels17NetworksOrder.nii.gz	Schaefer2018	1000p17n
+/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-200Parcels17NetworksOrder.nii.gz	Schaefer2018	p200n17
+/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-300Parcels17NetworksOrder.nii.gz	Schaefer2018	p300n17
+/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-400Parcels17NetworksOrder.nii.gz	Schaefer2018	p400n17
+/cpac_templates/Schaefer2018_space-FSLMNI152_res-2mm_desc-1000Parcels17NetworksOrder.nii.gz	Schaefer2018	p1000n17
```

Comment:
@gkiar I renamed the `desc`s on these atlases so the derivatives will be like `atlas-Schaefer2018p200n17` instead of like `atlas-Schaefer2018200p17n`, to avoid the chain of digits at the mergepoint

Reply:
That seems reasonable to me. Thanks!